### ISSUE TITLE
repository docker.io/archlinux-kdedev not found

### ISSUE THREAD

**Issue by bhootd** [Wenzel/docker-kdesrc-build#23]
Also, base/archlinux has been deprecated on Docker Hub, replaced by archlinux/base; but the dockerfile still points to the former.

Is the archlinux Dockerfile in obsolete state?

**Comment by Wenzel** [comment]
Hi,

I haven't been maintaining this repo since a few years, not a kde developer anymore.

I'm happy to merge PR though.

Otherwise you can fork and continue 🙂👍

**Comment by jcrben** [comment]
The neon one works.

**Comment by bhootd** [comment]
I did try neon first. But while building the plasma workspace, I encountered a copy error during cmake install where both target and destination were user-owned (so the error shouldn't have occurred).

Now while this error seems to be not related to this repo's neon docker image, I faced it only in that container. Later on, after fixing missing dependencies in Dockerfile-fedora, I didn't face this error, and the packages compiled and built successfully. So I have switched to Dockerfile-fedora.

However I do face another problem: I haven't been able to run the built-from-source plasma desktop by following the instructions given here: https://github.com/Wenzel/docker-kdesrc-build#1---using-a-shared-x11-socket

I face the error:

    startkde: Could not sync environment to dbus.

I am at my wit's end now trying to fix this. In case @Wenzel or @jcrben or really anyone else has any idea how to fix this, I could open another issue.

**Comment by bhootd** [comment]
I intend to create a pull request for Dockerfile-fedora once I get startkde to work successfully.

**Comment by jcrben** [comment]
@jayesh-bhoot have you asked for help anywhere else such as #kde-devel on freenode? I didn't end up finishing the entire neon build altho I dropped into the image, but I'll try to do that again. I was running it on a thinkpad t460s running ubuntu 18.04 lts.

I've also been messing around with building dolphin on mac (https://community.kde.org/Craft) and I think I've mostly figured it out. You can find me in #kde-mac on freenode.

**Comment by jcrben** [comment]
@jayesh-bhoot did you end up figuring this out? based on the stuff I've seen you saying at #kde-devel it seems like you figured something out - any wisdom to share with the rest of us? 😄

**Comment by bhootd** [comment]
@jcrben Yes I have finally managed to set things up, but using (and fixing a lot of) Fedora's Dockerfile.

I have been meaning to put everything together and create a pull request here. But it will take a while; the current Dockerfile has some instructions with static and personal data (like arcanist's auth token).

I should be able to wrap it up by this weekend.

**Comment by jcrben** [comment]
@jayesh-bhoot if you don't have time to make a PR, maybe you could describe some of the changes you had to do?
